spec/support/capybara_headers.rb

```diff
@@ -0,0 +1,17 @@
+require 'request_headers'
```

Comment:
remove this file

Reply:
Also the header of the table doesn't look aligned with the toolbar of the viewer. I know that they technically are, but is looks ugly.
![image](https://user-images.githubusercontent.com/327272/87438943-7260f080-c5f0-11ea-8016-ed90c5b2a84f.png)